app.rb

```diff
@@ -0,0 +1,70 @@
+require 'sinatra/base'
+require_relative 'models/user'
+require_relative 'models/peep'
+require_relative 'models/datamapper_setup'
+
+ENV['RACK_ENV'] ||= 'development'
+
+DataMapper.finalize
+DataMapper.auto_upgrade!
+
+class Chitter < Sinatra::Base
+
+  enable :sessions
+
+  get '/' do
+    redirect '/home'
+  end
+
+  get '/sign-up' do
+    erb :sign_up
+  end
+
+  post '/new-user' do
+    User.new_user_create(params[:name], params[:username], params[:email], params[:password])
+    session["username"] = params[:username] if User.redirect == '/home'
```

Comment:
It feels like the code in the User model that sets the value of User.redirect could probably be moved to the controller.  Deciding where to redirect the user is a messaging concern, rather than a data manipulation concern.